### PR TITLE
ci: rpm repository for all versions and aarch64

### LINKED ISSUE #3189: Install details don't work on Amazon Linux
## Description

Following https://aquasecurity.github.io/trivy/v0.34/getting-started/installation/#rhelcentos does not properly install Trivy on Amazon Linux

## What did you expect to happen?

Install Trivy

## What happened instead?

`https://aquasecurity.github.io/trivy-repo/rpm/releases/2/x86_64/repodata/repomd.xml: [Errno 14] HTTPS Error 404 - Not Found`

## Output of run with `-debug`:

n/a

## Output of `trivy -v`:

n/a

## Additional details (base image name, container registry info...):

```
$ cat /etc/os-release
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"
HOME_URL="https://amazonlinux.com/"
```

### PR DESCRIPTION
**Pull request by afdesk**
## Description
We had to set up a specific release version for RHEL/CentOS (ex. 5-9), so it didn't work correctly for Amazon Linux 2.
Now we can use the path without any versions.

Also `trivy-repo` adds ARM64 packages now.

before:
```
[trivy]
name=Trivy repository
baseurl=https://aquasecurity.github.io/trivy-repo/rpm/releases/$releasever/$basearch/
gpgcheck=1
enabled=1
gpgkey=https://aquasecurity.github.io/trivy-repo/rpm/public.key
```
after:
```
[trivy]
name=Trivy repository
baseurl=https://aquasecurity.github.io/trivy-repo/rpm/releases/$basearch/
gpgcheck=1
enabled=1
gpgkey=https://aquasecurity.github.io/trivy-repo/rpm/public.key
```
## Related issues
- Close #3189 

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: ci/deploy-rpm.sh [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+function create_common_rpm_repo () {
+        rpm_path=$1
+
+        ARCHES=("x86_64" "aarch64")
+        for arch in ${ARCHES[@]}; do
+                prefix=$arch
+                if [ "$arch" == "x86_64" ]; then
+                        prefix="64bit"
+                fi
+                if [ "$arch" == "aarch64" ]; then
```
Comment (realFranco) [inline ci/deploy-rpm.sh:14]: Hello, thank you for the contribution, seems to me that you could make use of the `elif` bash statement.

```suggestion
                elif [ "$arch" == "aarch64" ]; then
```

Comment (afdesk) [reply ci/deploy-rpm.sh:14]: @realFranco nice catch! thanks.

File: ci/deploy-rpm.sh [hunk 2]
```diff
@@ -18,6 +37,9 @@ echo "Create RPM releases for Trivy v$TRIVY_VERSION"
 
 cd trivy-repo
 
+echo "Processing common repository for RHEL/CentOS..."
+create_common_rpm_repo rpm/releases
```
[the remark this answers is not in the document]
Comment (afdesk) [reply ci/deploy-rpm.sh:41]: later we can remove versioned dirs

**Review by realFranco — APPROVED** [review]

File: ci/deploy-rpm.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (knqyf263) [reply ci/deploy-rpm.sh:41]: Thanks for confirming!

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**